### PR TITLE
Added rules to detect lolbas provlaunch.exe and also filter on legitimate system non-wmiprvse processes loading WMI modules

### PR DESCRIPTION
**Pull request by swachchhanda000**
### Summary of the Pull Request

Added rules to detect lolbas provlaunch.exe and also added filter on legitimate system non-wmiprvse processes loading WMI modules
### Detailed Description of the Pull Request / Additional Comments

1. Existing rules was creating too many fps results on even on legitimate processes on windows. Therefore filtering out fps.
2. There was no sigma rules attached for the lolbin 'Provlaunch'. SO adding the support
https://lolbas-project.github.io/lolbas/Binaries/Provlaunch/
### Example Log Event

N/A
### Fixed Issues

N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by swachchhanda000** [comment]
@nasbench , I have added new rules with this existing PR. Please review it

**Comment by nasbench** [comment]
Saw that :) Its in the pipeline

**Review by nasbench — APPROVED** [review]

File: rules/windows/process_creation/proc_creation_win_provlaunch_potential_abuse.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+id: 7f5d1c9a-3e83-48df-95a7-2b98aae6c13c
+related:
+    - id: f9999590-1f94-4a34-a91e-951e47bedefd # CLI Abuse
+      type: similar
+    - id: 2a4b3e61-9d22-4e4a-b60f-6e8f0cde6f25 # CLI Registry
+      type: similar
+    - id: 7021255e-5db3-4946-a8b9-0ba7a4644a69 # Registry
+      type: similar
+status: experimental
+description: Detect child processes of "provlaunch.exe" which might indicate potential abuse to proxy execution.
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_provlaunch_potential_abuse.yml:11]:
```suggestion
description: Detects child processes of "provlaunch.exe" which might indicate potential abuse to proxy execution.
```

File: rules/windows/process_creation/proc_creation_win_provlaunch_susp_child_process.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+id: f9999590-1f94-4a34-a91e-951e47bedefd
+related:
+    - id: 7f5d1c9a-3e83-48df-95a7-2b98aae6c13c # CLI Generic
+      type: similar
+    - id: 2a4b3e61-9d22-4e4a-b60f-6e8f0cde6f25 # CLI Registry
+      type: similar
+    - id: 7021255e-5db3-4946-a8b9-0ba7a4644a69 # Registry
+      type: similar
+status: experimental
+description: Detect suspicious child processes of "provlaunch.exe" which might indicate potential abuse to proxy execution.
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_provlaunch_susp_child_process.yml:11]:
```suggestion
description: Detects suspicious child processes of "provlaunch.exe" which might indicate potential abuse to proxy execution.
```

File: rules/windows/process_creation/proc_creation_win_registry_provlaunch_provisioning_command.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+id: 2a4b3e61-9d22-4e4a-b60f-6e8f0cde6f25
+related:
+    - id: 7f5d1c9a-3e83-48df-95a7-2b98aae6c13c # CLI Generic
+      type: similar
+    - id: f9999590-1f94-4a34-a91e-951e47bedefd # CLI Abuse
+      type: similar
+    - id: 7021255e-5db3-4946-a8b9-0ba7a4644a69 # Registry
+      type: similar
+status: experimental
+description: Detects potential abuse of the provisioning registry key abuse for indirect command execution through "Provlaunch.exe".
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_registry_provlaunch_provisioning_command.yml:11]:
```suggestion
description: Detects potential abuse of the provisioning registry key for indirect command execution through "Provlaunch.exe".
```

File: rules/windows/registry/registry_set/registry_set_provisioning_command_abuse.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+id: 7021255e-5db3-4946-a8b9-0ba7a4644a69
+related:
+    - id: 7f5d1c9a-3e83-48df-95a7-2b98aae6c13c # CLI Generic
+      type: similar
+    - id: f9999590-1f94-4a34-a91e-951e47bedefd # CLI Abuse
+      type: similar
+    - id: 2a4b3e61-9d22-4e4a-b60f-6e8f0cde6f25 # CLI Registry
+      type: similar
+status: experimental
+description: Detects potential abuse of the provisioning registry key abuse for indirect command execution through "Provlaunch.exe".
```
Comment (phantinuss) [inline rules/windows/registry/registry_set/registry_set_provisioning_command_abuse.yml:11]:
```suggestion
description: Detects potential abuse of the provisioning registry key for indirect command execution through "Provlaunch.exe".
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**